common/src/test/org/apache/hadoop/hive/conf/TestHiveConf.java

```diff
@@ -176,6 +176,20 @@ public void testHiddenConfig() throws Exception {
     }
   }
 
+  @Test
+  public void testLockedConfig() throws Exception {
+    HiveConf conf = new HiveConf();
+    // Get the default value of the config
+    String defaultVal = conf.get("hive.execution.engine");
```

Comment:
I'd say we should explicitly set `hive.execution.engine=mr` here because it could be possible that tez will be the default engine someday.

Reply:
Sure, will update the test.
Thanks